base/uk.ac.stfc.isis.ibex.opis/resources/polaris_jaws/Jaws_manager.opi

```diff
@@ -4,7 +4,7 @@
   <rules />
   <wuid>7f94559c:141357de30e:-7fc5</wuid>
   <show_grid>true</show_grid>
-  <auto_zoom_to_fit_all>false</auto_zoom_to_fit_all>
+  <auto_zoom_to_fit_all>true</auto_zoom_to_fit_all>
```

Comment:
I'm not convinced by this - I see what you're doing and it is a nice option, but it can get the OPI into some unreadable states, which a user might not be able to get out of if they don't know that the OPI can be resized...

![image](https://cloud.githubusercontent.com/assets/24375813/25612128/6f643fd6-2f21-11e7-86a5-16842b384cc7.png)

At the default size of a device screens OPI the font is already quite small. I'm just concerned that they will complain it's unreadable without realising it auto-scales.